diff --git a/web/modules/custom/last_haiku_import/last_haiku_import.install b/web/modules/custom/last_haiku_import/last_haiku_import.install
index 067b6ef..7d44819 100755
--- a/web/modules/custom/last_haiku_import/last_haiku_import.install
+++ b/web/modules/custom/last_haiku_import/last_haiku_import.install
@@ -2794,13 +2794,19 @@ function last_haiku_import_make_redirects() {
 
      foreach( $clans as $clan ) {
 
-          $redirects[ 'families/' . strtolower( $clan ) ] = '/families?clan[]=' . last_haiku_import_term_lookup( $clan, 'clans' );
+          $clanTerm = last_haiku_import_term_lookup( $clan, 'clans' );
+
+          $redirects[ 'families/' . strtolower( $clan ) ] = '/families?clan[]=' . $clanTerm;
 
           if( $clan == 'Ronin Families' ) {
                $clan = 'Ronin Schools';
           }
 
-          $redirects[ 'schools/' . strtolower( $clan ) ] = '/schools?clan[]=' . last_haiku_import_term_lookup( $clan, 'clans' );
+          $redirects[ 'schools/' . strtolower( $clan ) ] = '/schools?clan[]=' . $clanTerm;
+
+          $redirects[ 'schools/' . strtolower( $clan ) . '/basic' ] = '/schools?clan[]=' . $clanTerm . '&type=schools';
+          $redirects[ 'schools/' . strtolower( $clan ) . '/advanced' ] = '/schools?clan[]=' . $clanTerm . '&type=advanced_schools';
+          $redirects[ 'schools/' . strtolower( $clan ) . '/paths' ] = '/schools?clan[]=' . $clanTerm . '&type=alternate_paths';
      }
 
      $redirects[ 'families/falcon' ] = '/families?clan[]=' . last_haiku_import_term_lookup( 'Crab', 'clans' ) . '#Toritaka';
diff --git a/web/themes/custom/km/build/css/main.style.css b/web/themes/custom/km/build/css/main.style.css
index 5ec5d24..e82db41 100755
--- a/web/themes/custom/km/build/css/main.style.css
+++ b/web/themes/custom/km/build/css/main.style.css
@@ -12697,6 +12697,7 @@ main.py-5 {
 
 .navbar-expand-lg.fixed-top {
   z-index: 1200;
+  max-height: 100vh;
 }
 @media (min-width: 992px) {
   .navbar-expand-lg {
@@ -12713,7 +12714,7 @@ main.py-5 {
   display: block;
 }
 @media (min-width: 992px) {
-  .navbar-expand-lg .navbar-nav li:hover .dropdown-menu {
+  .navbar-expand-lg .navbar-nav li:hover > .dropdown-menu {
     display: block;
   }
 }
diff --git a/web/themes/custom/km/src/scss/theme/nav.scss b/web/themes/custom/km/src/scss/theme/nav.scss
index 6e9049b..410daa1 100644
--- a/web/themes/custom/km/src/scss/theme/nav.scss
+++ b/web/themes/custom/km/src/scss/theme/nav.scss
@@ -1,6 +1,7 @@
 .navbar-expand-lg {
      &.fixed-top {
           z-index: 1200;
+          max-height: 100vh;
      }
 
      @include media-breakpoint-up(lg) {
@@ -27,7 +28,7 @@
 
                @include media-breakpoint-up(lg) {
                     &:hover {
-                         .dropdown-menu {
+                         & > .dropdown-menu {
                               display: block;
                          }
                     }
